content/kubernetes/re-clusters/upgrade-redis-cluster.md

```diff
@@ -78,62 +83,96 @@ customresourcedefinition.apiextensions.k8s.io/redisenterprisedatabases.app.redis
 deployment.apps/redis-enterprise-operator configured
 ```
 
-### Verify that the operator is running
+### Reapply other manual configurations
+
+When upgrading your cluster, you need to manually reapply the `ValidatingWebhookConfiguration` for the admission controller. See the [Enable the admission controller]({{<relref "/kubernetes/deployment/quick-start#enable-the-admission-controller">}}) step during deployment for more details.
 
-You can verify the operator is running in your namespace by checking the deployment as follows:
 
+If you are using Openshift, you will also need to manually reapply the [Security context constraint](https://docs.openshift.com/container-platform/4.8/authentication/managing-security-context-constraints.html) file ([`scc.yaml`]({{<relref "/kubernetes/deployment/openshift/openshift-cli#deploy-the-operator" >}})).
+
+```sh
+oc apply -f openshift/scc.yaml
 ```
+
+### Verify the operator is running
+
+You can check your deployment to verify the operator is running in your namespace.
+
+```sh
 kubectl get deployment/redis-enterprise-operator
 ```
 
 You should see a result similar to this:
 
-```
+```sh
 NAME                        READY   UP-TO-DATE   AVAILABLE   AGE
 redis-enterprise-operator   1/1     1            1           0m36s
 ```
 
-### 2- Upgrade the Redis Enterprise cluster version
+{{< warning >}}
+ We recommend upgrading the REC as soon as possible after updating the operator. After the operator upgrade completes, the REC is considered frozen. The operator will no longer manage the REC and the REC and its REDB can't be modified until the REC upgrade completes.
```

Comment:
`updating the operator` --> `upgrading the operator`.

Also, I'm not sure "frozen" should be used, it's more of an internal term (although there's a user-visible event fired with `reason: UpgradeFreeze` but still...).
Maybe focus on the implications, something along the lines of `After the operator upgrade completes, the operator suspends the management of the REC and its associated REDBs, until the REC upgrade completes`.